docs/source/project_documentation/user_guide/usecase_definitions.md

```diff
@@ -0,0 +1,59 @@
+# Usecase Definitions in AaC
+
+One of AaC's goal is to allow for a system to be modeled in a way that various disciplined users would be able to understand and implement the model for their contribution to the system. One of the system model aspects captured within AaC is the behavior of the system, which has a rootkey of `usecase`. The `usecase` describes a system behavior, who is involved with the system behavior, and outlines the steps for achieving the described system behavior.
+
+## Utilizing Usecase Definitions
+
+There is an example usecase within our alarm clock model example for setting the alarm time.
+
+```{eval-rst}
+.. literalinclude:: ../../../../python/model/alarm_clock/usecase.yaml
+    :language: yaml
+    :lines: 6-24
+```
+By incorporating the `usecase`s for various system behaviors, the alarm clock is able to be evaluated for functional completeness that cannot be seen in the alarm clock model alone.
+
+Some questions that can when evaluating the system for functional completeness are below. They can be addressed with the incorporation of a `usecase` definition for the system.
+
+1. Does the system allow for necessary participants?
```

Comment:
Might just be a limit of my understanding, but allowing for a model seems like an ambiguous alternate of 'define'.
```suggestion
1. Does the system define the necessary participants?
```